packages/trader/src/AppV2/Components/RiskManagementItem/risk-management-item.tsx

```diff
@@ -116,13 +116,17 @@ const RiskManagementItem = observer(
                         ))}
                     {is_valid_to_cancel && is_deal_cancellation && <DealCancellationRemainingTime />}
                 </div>
-                {!is_accumulator && isToggleOn && (
+                {!is_accumulator && isToggleOn && currency && (
                     <TextField
                         variant='fill'
                         inputSize='md'
                         disabled={isSheetOpen}
                         textAlignment='center'
-                        value={`${finalValue.toFixed(2)} ${currency}`}
+                        value={`${
+                            type == 'stop_loss'
+                                ? `-${formatMoney(currency, finalValue, true)} ${currency}`
+                                : `${formatMoney(currency, finalValue, true)} ${currency}`
+                        }`}
```

Comment:
```suggestion
                        value={`${formatMoney(currency, type == 'stop_loss' ? -finalValue: finalValue, true)} ${currency}`}
```
formatMoney fn can handle the sign too

Reply:
done thanks